doobie/src/main/scala/tamer/db/ConfigDb.scala

```diff
@@ -0,0 +1,44 @@
+package tamer.db
```

Comment:
```suggestion
package tamer
package db
```

Reply:
I've seen this done often, but, doesn't this actually go against the principle of least privilege?